CoreWelcomeView/src/au/gov/asd/tac/constellation/views/welcome/WelcomeTopComponent.java

```diff
@@ -0,0 +1,360 @@
+
+    public WelcomeTopComponent() {
+        setName(Bundle.CTL_WelcomeTopComponentTopComponent());
+        setToolTipText(Bundle.HINT_WelcomeTopComponentTopComponent());
+        setLayout(new BorderLayout());
+
+        ProxyUtilities.setProxySelector(null);
+
+        // Add the JavaFX container to this topcomponent (this enables JavaFX and Swing interoperability):
+        add(panel, BorderLayout.CENTER);
+
+        ConstellationSecurityManager.startSecurityLaterFX(() -> {
+            Platform.setImplicitExit(false);
+
+            //The root swing object to be inseted into a JFX Panel
+            final BorderPane root = new BorderPane();
+
+            final SplitPane splitPane = new SplitPane();
+            splitPane.setOrientation(Orientation.HORIZONTAL);
+
+            splitPane.setStyle("-fx-background-color: transparent;");
+            root.setCenter(splitPane);
+
+            //Create VBox to handle Browser and controls,
+            //or error messages
+            final VBox leftVBox = new VBox();
+            splitPane.getItems().add(leftVBox);
+            leftVBox.setSpacing(20);
+            leftVBox.setMinWidth(350);
+            leftVBox.setPrefWidth(400);
+            leftVBox.setMaxWidth(450);
+
+            final HBox logoHBox = new HBox();
+            logoHBox.setBackground(new Background(new BackgroundFill(Color.valueOf("white"), CornerRadii.EMPTY, Insets.EMPTY)));
+            ImageView logoView = new ImageView(new Image(WelcomeTopComponent.class.getResourceAsStream(LOGO)));
+            logoView.setFitHeight(100);
+            logoView.setFitWidth(150);
+            logoHBox.getChildren().add(logoView);
+            logoHBox.setAlignment(Pos.CENTER);
+            leftVBox.getChildren().add(logoHBox);
+
+            //Create the labels for the left pane
+            final Label welcome = new Label(WELCOME_TEXT);
+            welcome.setFont(new Font("Arial Unicode MS", 26));
+            welcome.setAlignment(Pos.CENTER);
+            leftVBox.getChildren().add(welcome);
+
+            //Create right VBox for graph controls
+            final VBox rightVBox = new VBox();
+            rightVBox.setPadding(new Insets(50, 50, 50, 50));
+            rightVBox.setBackground(new Background(new BackgroundFill(Color.valueOf("#14161a"), CornerRadii.EMPTY, Insets.EMPTY)));
+            splitPane.getItems().add(rightVBox);
+
+            //Create HBoxes for the right_vbox
+            final HBox topHBox = new HBox();
+            final HBox bottomHBox = new HBox();
+
+            //hbox formatting
+            topHBox.setPadding(new Insets(50, 0, 50, 0));
+            topHBox.setSpacing(10);
+            bottomHBox.setPadding(new Insets(50, 0, 50, 0));
+            bottomHBox.setSpacing(10);
+
+            getWelcomeContent();
+
+            for (int i = 0; i < 10; i++) {
+                if (pluginButtons[i] != null) {
+                    Button currentButton = pluginButtons[i];
```

Comment:
I believe it is pass by reference rather than by value. If that is indeed the case, I would either replace subsequent calls on `pluginButtons[i]` with `currentButton` (and make `currentButton` final) or remove `currentButton` and replace subsequent calls of it with `pluginButtons[i]`